resources/core/charts/service-catalog-addons/README.md

```diff
@@ -0,0 +1,27 @@
+# Service Catalog Addons
+
+## Overview
+
+Service Catalog Addons provides Kyma addons to the [Service Catalog](../service-catalog/README.md).
+
+These addons consist of new views related to the Service Catalog and the Binding Usage Controller which extends the Service Catalog with the additional logic. 
+
+### Views
```

Comment:
The section as a whole is a bit short. I wonder whether we need the heading at all. We can just skip it and start with such a sentence:
`These are the Kyma Console views:`

And then we can start straight with the BUC section. WDYT?